### PR TITLE
Version 0.2.0

### PR DESCRIPTION
**Pull request by tamirkamara**
Include Kusto SDK 1.0.0-BETA-01, include some tests, fix password bug

### CODE REVIEW

File: lib/logstash/outputs/kusto/ingestor.rb [hunk 1]
```diff
@@ -29,11 +29,11 @@ def initialize(ingest_url, app_id, app_key, app_tenant, database, table, mapping
 
       @logger.debug('Preparing Kusto resources.')
 
-      kusto_connection_string = Java::KustoConnectionStringBuilder.createWithAadApplicationCredentials(ingest_url, app_id, app_key, app_tenant)
+      kusto_connection_string = Java::com.microsoft.azure.kusto.data.ConnectionStringBuilder.createWithAadApplicationCredentials(ingest_url, app_id, app_key.value, app_tenant)
```
Comment (i3arnon) [inline lib/logstash/outputs/kusto/ingestor.rb]: There's extra whitespace at the end (if you care about it)

File: lib/logstash/outputs/kusto/ingestor.rb [hunk 2]
```diff
@@ -90,7 +90,8 @@ def upload(path, delete_on_success)
       #   local_ingestion_properties.addJsonMappingName(mapping)
       # end
 
-      @kusto_client.ingestFromSingleFile(path, @ingestion_properties)
+      file_source_info = Java::com.microsoft.azure.kusto.ingest.source.FileSourceInfo.new(path, 0); # 0 - let the sdk figure out the size of the file
```
Comment (i3arnon) [inline lib/logstash/outputs/kusto/ingestor.rb]: Unrelated, but wouldn't we want to add an overload that doesn't accept the file size to the Java client?

File: logstash-output-kusto.gemspec [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-  s.version       = '0.1.6'
+  s.version       = '0.2.0'
   s.licenses      = ['Apache-2.0']
-  s.summary       = 'Writes events to Azure KustoDB'
-  s.description   = 'This is a logstash output plugin used to write events to an Azure KustoDB instance'
-  s.homepage      = 'https://github.com/Azure/azure-diagnostics-tools/tree/master/Logstash'
+  s.summary       = 'Writes events to Azure Data Explorer (Kusto)'
+  s.description   = 'This is a logstash output plugin used to write events to an Azure Data Explorer (a.k.a Kusto)'
+  s.homepage      = 'https://github.com/Azure/logstash-output-kusto'
   s.authors       = ['Tamir Kamara']
```
Comment (i3arnon) [inline logstash-output-kusto.gemspec]: Do you still want your name there? (because you removed your email)

Comment (tamirkamara) [reply logstash-output-kusto.gemspec]: I don't mind the name...

File: lib/logstash/outputs/kusto/ingestor.rb (hunk 2 above)
Comment (tamirkamara) [reply lib/logstash/outputs/kusto/ingestor.rb]: Yes, I guess we can do that. Do you have it on other SDKs (Python)?

**Pull request merged**